conda-env.yml

```diff
@@ -6,21 +6,8 @@ dependencies:
   - cython
   - ddt
   - gitdb
-  - gitpython
-  - lz4
-  - matplotlib
-  - numpy
   - opencv
-  # PILLOW_VERSION removed which causes torchvision<=0.4.0 to fail
-  - pillow<7
   - pip
-  - pytorch<1.3
-  - pyyaml
-  - scikit-learn
-  - six
-  - torchvision
-  - tqdm
   - pip:
-    - h5py
-    - opencv-python
     - tensorboardX
+    - -r requirements.txt
```

Comment:
Did not know this was possible at all, you are truly a hacker

Reply:
I just suspected conda was forwarding arguments to pip command. Gave it a try and lucky us it works. :)